CI-testing failed because of `protobuf`

### Issue description

I notice that we're having some tasks failed in our CI-testing workflow, for example, https://github.com/WenjieDu/PyPOTS/actions/runs/5064524248/jobs/9092539446.

``` bash
/usr/share/miniconda/envs/pypots-test/lib/python3.8/site-packages/google/protobuf/descriptor.py:51: in <module>
    from google.protobuf.pyext import _message
E   ImportError: /usr/share/miniconda/envs/pypots-test/lib/python3.8/site-packages/google/protobuf/pyext/_message.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN6google8protobuf2io17SafeDoubleToFloatEd
```

I have investigated this error and it has nothing to do with code in PyPOTS. The issue source is that these failed jobs have their conda installing a newer version of protobuf which is not compatible with other dependencies. It can be directly solved by specifying the version of protobuf as `4.21.12`, and I have tested it.

--- a/tests/environment_for_conda_test.yml
+++ b/tests/environment_for_conda_test.yml
@@ -18,6 +18,7 @@ dependencies:
     - conda-forge::tensorboard
     - conda-forge::pycorruptor
     - conda-forge::tsdb
+    - conda-forge::protobuf ==4.21.12
     - pytorch::pytorch >=1.10.0
 
     # optional

--- a/tests/environment_for_conda_test.yml
+++ b/tests/environment_for_conda_test.yml
@@ -18,7 +18,7 @@ dependencies:
     - conda-forge::tensorboard
     - conda-forge::pycorruptor
     - conda-forge::tsdb
-    - conda-forge::protobuf ==4.21.12
+    - conda-forge::protobuf <=4.21.12
     - pytorch::pytorch >=1.10.0
 
     # optional

